sdcm/provision/scylla_yaml/scylla_yaml.py

```diff
@@ -342,6 +342,8 @@ def set_authorizer(cls, authorizer: str):
 
     compaction_collection_items_count_warning_threshold: int = None  # None
 
+    enable_tablets: bool = None  # False, but default scylla.yaml for some versions (e.g. 6.0) override it to True
```

Comment:
Value is set to "None", but comment says "False"?

Reply:
Comment refers to scylla specific versions that have defaults for it.

None means we do mess with scylla default

Reply:
It is what comments are needed for - reduce unclarity and confusion. Not increasing it.

> None means we do mess with scylla default

Does it have some typo or you explicitly state what is written?

Reply:
this is an uncommon case: when Scylla defaults differ from scylla.yaml template shipped.
I added a comment for that, as typically when reviewing this class, reader could think that tablets are kept default.
And None means we don't touch it when constructing scylla.yaml out of existing one.